diff -Naur gcc-python-plugin-0.12.orig/gcc-with-python gcc-python-plugin-0.12/gcc-with-python
--- gcc-python-plugin-0.12.orig/gcc-with-python	2014-04-21 21:25:05.000000000 +0400
+++ gcc-python-plugin-0.12/gcc-with-python	2014-04-21 21:26:08.000000000 +0400
@@ -16,4 +16,4 @@
 #   along with this program.  If not, see
 #   <http://www.gnu.org/licenses/>.
 
-${CC:-gcc} -fplugin=$(pwd)/python.so -fplugin-arg-python-script=$@
+${CC:-gcc} -fplugin=$(gcc -print-file-name=plugin)/python.so -fplugin-arg-python-script=$@
